meta-balena-common/recipes-containers/hostapp-extensions-update/files/update-hostapp-extensions

```diff
@@ -63,18 +61,18 @@ umount_merged() {
 	fi
 }
 
-# The target hostapp extensions is a space separated list of docker repository
-# tags in the form repo/name:tag, e.g balena/default:v1 balena/extra:v2
+# The target hostapp extensions is a space separated list of Balena
+# releases, e.g  <repository>/<image>@<digest>
 #
 # Select target hostapp extensions in order of preference:
 # Command line argument
-# hostappExtensions entry in config.json
-# Image default from /etc/hostapp-extensions.conf
+# Preloaded images from apps.json
+RESIN_DATA_DIR="/mnt/data"
+APPS_JSON_PATH="${RESIN_DATA_DIR}/apps.jon"
```

Comment:
There is a typo here

Reply:
s/apps.jon/apps.json/

Reply:
Fixed, thanks. I still haven't able to completely setup the devenv to test the script, so it's good you have :)